tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -176,6 +180,75 @@
                        src=src, subjects_dir=subjects_dir, dig=True,
                        surfaces=['head-dense', 'white'], coord_frame='meg')
 
+###############################################################################
+# Visualizing the transformation of digitized points
+# --------------------------------------------------
+# Let's step through the process that the transform is accomplishing.
+#
+# First, let's define a helper function to plot the alignment of the
+# points with the head.
+
+
+def plot_dig_alignment(points, coord_system='ras'):
+    points = points.copy()
+    if coord_system == 'mri':  # transform back to RAS
+        points = ((points - 128) * [1, -1, 1])[:, [0, 2, 1]]
+    else:  # m → mm
+        points *= 1e3
+    renderer = mne.viz.backends.renderer._get_renderer(
+        size=(800, 400), bgcolor='w')
+    seghead_rr, seghead_tri = mne.read_surface(
+        op.join(subjects_dir, 'sample', 'surf', 'lh.seghead'))
+    renderer.mesh(*seghead_rr.T, triangles=seghead_tri, color=(0.7,) * 3,
+                  opacity=0.2)
+    for point in points:
+        renderer.sphere(center=point, color='r', scale=5)
+    mne.viz.set_3d_view(figure=renderer.figure, distance=1000,
+                        focalpoint=(0., 0., 0.), elevation=90, azimuth=0)
+    renderer.show()
+
+
+###############################################################################
+# Plot untransformed head space digitized points as if they were in RAS space
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# The plot below shows the digitization points in the coordinate space
+# native to the digitizer equipment plotted with the head surface in
+# right-anterior-superior (RAS) coordinates. The relationship between the
```

Comment:
Here again: as far as the naive learner knows, all 3 coord systems are RAS. So it's not clear which coordinate frame is being used to define the plot origin.